app-modules/docs/src/Discovery/Strategies/GuideStrategy.php

```
@@ -0,0 +1,39 @@
+    {
+        return DocumentType::Guide;
+    }
+
+    public function matches(SplFileInfo $file): bool
+    {
+        return str_contains($this->path($file), '/resources/docs/')
+            && str_ends_with($file->getFilename(), '.md')
+            && $file->getFilename() !== 'documentation.md';
+    }
+
+    protected function slug(SplFileInfo $file, DocumentMetadata $meta, ?string $module): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$meta'. (undefined)

(UnusedFormalParameter)